daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -176,7 +176,7 @@ func (l *logStream) Log(msg *logger.Message) error {
 	defer l.lock.RUnlock()
 	if !l.closed {
 		// buffer up the data, making sure to copy the Line data
-		l.messages <- logger.CopyMessage(msg)
+		l.messages <- msg
```

Comment:
Yes, I can remove this comment as it's no longer neccessary after the 2nd commit.